requirements_test.txt

```diff
@@ -1,5 +1,3 @@
--r requirements.txt
```

Comment:
why this change? to make the tox setup easier?

Reply:
Yeah — as the only thing in `requirements.txt` is `awesome-slugify`, which is now optional, it made sense to me to not include `requirements.txt` by default. 

an alternative would be to remove `awesome-slugify` from `requirements.txt` and explicitly include it in the deps in `tox.ini` — but then, `requirements.txt` will be an empty file and it wouldn't make sense to include it in `requirements_test.txt` anyway.

happy to change if you wish!

Reply:
ah I see, no problem then all good